x/ccv/provider/keeper/keeper_test.go

```diff
@@ -369,28 +369,28 @@ func TestVscSendTimestamp(t *testing.T) {
 	}
 
 	i = 0
-	providerKeeper.IterateVscSendTimestamps(ctx, testCases[0].chainID, func(vscID uint64, ts time.Time) bool {
+	providerKeeper.IterateVscSendTimestamps(ctx, testCases[0].chainID, func(vscID uint64, ts time.Time) (stop bool) {
 		require.Equal(t, vscID, testCases[i].vscID)
 		require.Equal(t, ts, testCases[i].ts)
 		i++
-		return true
+		return false // do not stop
 	})
 	require.Equal(t, 2, i)
 
 	// delete VSC send timestamps
 	var ids []uint64
-	providerKeeper.IterateVscSendTimestamps(ctx, testCases[0].chainID, func(vscID uint64, _ time.Time) bool {
+	providerKeeper.IterateVscSendTimestamps(ctx, testCases[0].chainID, func(vscID uint64, _ time.Time) (stop bool) {
```

Comment:
Don't need to iterate here twice, but that's not important for this PR

Reply:
The two calls to `IterateVscSendTimestamps` as they exist could be replaced with one iterator
```go
	var ids []uint64
	i = 0
	providerKeeper.IterateVscSendTimestamps(ctx, testCases[0].chainID, func(vscID uint64, ts time.Time) (stop bool) {
		ids = append(ids, vscID)
		require.Equal(t, vscID, testCases[i].vscID)
		require.Equal(t, ts, testCases[i].ts)
		i++
		return false // do not stop
	})
	require.Equal(t, 2, i)
	for _, vscID := range ids {
		providerKeeper.DeleteVscSendTimestamp(ctx, testCases[0].chainID, vscID)
	}
```

Reply:
Oh, yeah. Not important though.